compiler/modules/front.syntax/tests/testParser.bal

```diff
@@ -549,3 +549,11 @@ function canonFileName(string base) returns string{
     string sansExt = base.substring(0, base.length() - SOURCE_EXTENSION.length());
     return sansExt + "-canon" + SOURCE_EXTENSION;
 }
+
+function locationFromFilename(string filename) returns err:Location {
```

Comment:
minor: I think just `fromFilename` is sufficient.

Reply:
fixed

Reply:
Not sufficient.  How does fromFilename communicate what you are doing with a filename?  langlib uses fromT only when the module is dealing with a single type e.g. string:fromInt.

Reply:
Hmm OK, anyway I felt that it was too long. Another option is to make it just `location(string filename)`

Reply:
I agree locationFromFilename is a bit on the long side: location is fine too.

Reply:
fixed